.gitignore

```diff
@@ -34,6 +34,9 @@ m4/ltversion.m4
 m4/lt~obsolete.m4
 src/H5Edefin.h
 src/H5Einit.h
+src/H5E.h
```

Comment:
Was this file added somewhere? I didn't seem to see it in the perl changes

Reply:
> Was this file added somewhere? I didn't seem to see it in the perl changes

Typo